faunadb-java-dsl/src/main/java/com/faunadb/client/util/Objects.java

```diff
@@ -0,0 +1,27 @@
+package com.faunadb.client.util;
+
+import java.util.Arrays;
+
+final public class Objects {
```

Comment:
Since this is public, I think we need to document it. Specially because people will wonder why we didn't use the jdk native methods.